### PR TITLE
Creating FPN ToS

### PR DESCRIPTION
**Pull request by litkaemily**
cc @mafeld for viz

@peiying2 , please LMK if you see any issues. Thanks in advance!

### CODE REVIEW

File: Firefox_Private_Network_ToS/en-US.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# Firefox Private Network Browser Extension (Beta) Terms of Service
+
+Version 3.0, effective July 7, 2020
```
Comment (peiying2) [inline Firefox_Private_Network_ToS/en-US.md:3]: Use the same page title and date format like the one you did: https://github.com/mozilla/legal-docs/pull/1463/files

File: Firefox_Private_Network_ToS/en-US.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+Mozilla is a global organization, and our computers and service providers’ computers may be in various countries around the world, including the United States. This means that your information might be processed on servers located outside of the country where you live, and that country may have a different level of data protection regulation than yours. By giving us information, you consent to this kind of transfer of your information.
+
+## Permissions Necessary to Provide the Service
+
+Firefox Private Network is in partnership with Cloudflare. To learn more about how the Service works, you can see Mozilla’s [source code](https://github.com/mozilla/secure-proxy) for Firefox Private Network.
+
+__Your Use of the Service.__ You give Mozilla and Cloudflare all rights to operate the Service, including processing the data you upload to or access through the Service. You can find more information about the data Mozilla and Cloudflare process in the Firefox Private Network [Privacy Notice](https://www.mozilla.org/privacy/firefox-private-network/).
```
Comment (peiying2) [inline Firefox_Private_Network_ToS/en-US.md:25]: Include the full FPN PN in the link: `[Firefox Private Network Privacy Notice](...), like in line 17.

File: Firefox_Private_Network_ToS/en-US.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+Mozilla gives you permission to use the Service according to these Terms of Service. This permission is only for your personal use. You may not transfer, sublicense, resell, or commercially exploit the service.  
+
+You also agree that your use of the Service will comply with Mozilla’s [Conditions of Use](https://www.mozilla.org/about/legal/acceptable-use/). You are solely responsible for the Content you transmit and the consequences.
```
Comment (peiying2) [inline Firefox_Private_Network_ToS/en-US.md:29]: Do you want to include `Mozilla's` as part of the `[Mozilla’s Conditions of Use]`? It's up to you.

File: Firefox_Private_Network_ToS/en-US.md (hunk 1 above)
Comment (litkaemily) [reply Firefox_Private_Network_ToS/en-US.md:3]: We need to be more consistent about this. Let's keep effective. I'll update the others to match this standard.

File: Firefox_Private_Network_ToS/en-US.md (hunk 2 above)
Comment (litkaemily) [reply Firefox_Private_Network_ToS/en-US.md]: Updated

File: Firefox_Private_Network_ToS/en-US.md (hunk 3 above)
Comment (litkaemily) [reply Firefox_Private_Network_ToS/en-US.md:29]: This is fine. But, you make a good point. We need to standardize these. I'll work on a style document.

**Pull request merged**